root/edit/details/EditWork.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * @flow
+ * Copyright (C) 2020 Anirudh Jain
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import WordDiff from '../../static/scripts/edit/components/edit/WordDiff';
+import {INSERT, DELETE} from '../../static/scripts/edit/utility/editDiff';
+import Diff from '../../static/scripts/edit/components/edit/Diff';
+import DiffSide from '../../static/scripts/edit/components/edit/DiffSide';
+import FullChangeDiff from
+  '../../static/scripts/edit/components/edit/FullChangeDiff';
+import {commaOnlyListText} from
+  '../../static/scripts/common/i18n/commaOnlyList';
+
+type Props = {
+  ...EditT,
+  +data: {
+    +new: {
+      +attributes?: {
+        +[attributeName: string]: $ReadOnlyArray<WorkAttributeT>,
+      },
+      +languages: $ReadOnlyArray<string>,
+    },
+  },
```

Comment:
This part can be removed pending https://github.com/metabrainz/musicbrainz-server/pull/1397/files#r382935981